src/models/User.ts

```diff
@@ -19,6 +19,9 @@ export class User {
     @Column({name: "fullName", nullable: true})
     public fullName: string | undefined;
 
+    @Column({name: "notifications", nullable: true})
```

Comment:
You could also give notifications a default value instead of making it nullable.
I think setting the default value to false would be better.